packages/example-getting-started/README.md

```diff
@@ -0,0 +1,419 @@
+# @loopback/example-getting-started
+
+This is the basic tutorial for getting started with Loopback 4!
+
+**NOTICE**: This tutorial is currently under construction! This notice will be
+removed when it is ready for use!
+
+## Prerequisites
+
+Before we can begin, you'll need to make sure you have some things installed:
+- [Node.js](https://nodejs.org/en/) at v6.x or greater
+- [TypeScript](http://www.typescriptlang.org/)
+- [Git](https://git-scm.com/)
```

Comment:
Git should not be needed, because people are going to obtain this example via `lb4 example` command.

Please update the instructions accordingly.

Reply:
@kjdelisle PTAL ☝️  this hasn't been addressed yet.